docs/app/Examples/elements/Header/Content/HeaderIconExample.js

```diff
@@ -0,0 +1,15 @@
+import React, {Component} from 'react';
+import {Header} from 'stardust';
+
+export default class HeaderIconExample extends Component {
+  render() {
+    const plug = <i className='plug icon' />;
+    return (
+      <Header.H2 icon={plug}>
+        <div className='content'>
```

Comment:
Do you mean updating the component so this could be written as `<Header.H2 icon={plug} content="Is Your Electricity Truly Electrifying?" />`?